templates/tftmpl/notifier/consul_kv_test.go

```diff
@@ -195,51 +147,17 @@ func Test_ConsulKV_Notify_Once_Mode_Single_Key(t *testing.T) {
 		assert.True(t, n.once, "got 3/3 deps. once-mode should be completed")
 		assert.Equal(t, 3, n.counter, "second services should be 3rd dep")
 
-		// check mock template was called twice
+		// check mock template was called three times
 		tmpl.AssertExpectations(t)
 	})
 
-	t.Run("consul-kv-last-with-existing-key", func(t *testing.T) {
+	t.Run("consul-kv-last", func(t *testing.T) {
 		// Test that notifier notifies at the end of once-mode, particularly for
 		// the case when the consul-kv dependency is received last.
 
-		// Notifier in test has 3 dependencies: 1 services, 1 Consul KV exists, 1 Consul KV value
+		// Notifier in test has 2 dependencies: 1 services, 1 Consul KV exists
```

Comment:
I guess the dependencies are actually 1 services, 1 Consul KV? probably don't need to specify exists.